testing/src/scenario/state.py

```diff
@@ -490,7 +490,7 @@ def _next_relation_id(*, update: bool = True):
 
 @dataclasses.dataclass(frozen=True)
 class RelationBase(_max_posargs(2)):
-    """Base class for the various types of integration (relation)."""
+    """Base class for the various types of relation (integration)."""
```

Comment:
I'd just drop integration.

```suggestion
    """Base class for the various types of relation."""
```